sentry_sdk/ai/monitoring.py

```diff
@@ -106,14 +106,14 @@ def record_token_usage(
     if ai_pipeline_name:
         span.set_attribute("ai.pipeline.name", ai_pipeline_name)
     if prompt_tokens is not None:
-        span.set_measurement("ai_prompt_tokens_used", value=prompt_tokens)
+        span.set_attribute("ai.prompt_tokens_used", prompt_tokens)
     if completion_tokens is not None:
-        span.set_measurement("ai_completion_tokens_used", value=completion_tokens)
+        span.set_attribute("ai.completion_tokens_used", completion_tokens)
     if (
         total_tokens is None
         and prompt_tokens is not None
         and completion_tokens is not None
     ):
         total_tokens = prompt_tokens + completion_tokens
     if total_tokens is not None:
-        span.set_measurement("ai_total_tokens_used", total_tokens)
+        span.set_attribute("ai.total_tokens_used", total_tokens)
```

Comment:
[change request] span convention

```suggestion
        span.set_attribute("ai.total_tokens.used", total_tokens)
```